pkg/attacher/connection.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package connection
```

Comment:
`connection` was thin layer around gRPC connection to a real driver so it can be mocked out in unit tests. IMO, `attacher` is IMO something else.